internal/worker/worker.go

```diff
@@ -196,6 +206,51 @@ func (worker *Worker) Run(ctx context.Context) error {
 	}
 }
 
+func (worker *Worker) tryCreateStandby(ctx context.Context) {
+	// Do nothing if no standby instance is configured
+	if worker.standbyIsolation == nil {
+		return
+	}
+
+	// Do nothing if the standby instance is already instantiated
+	if worker.standbyInstance != nil {
+		return
+	}
+
+	// Do nothing if there are tasks that are running to simplify the resource management
+	if worker.tasks.Size() != 0 {
+		return
+	}
+
+	worker.logger.Debugf("creating a new standby instance with isolation %s", worker.standbyIsolation)
+
+	standbyInstance, err := persistentworker.New(worker.standbyIsolation, worker.security, worker.logger)
+	if err != nil {
+		worker.logger.Errorf("failed to create a standby instance: %v", err)
+
+		return
+	}
+
+	logger := echelon.NewLogger(echelon.TraceLevel, renderers.NewSimpleRenderer(os.Stdout, nil))
+
+	worker.logger.Debugf("warming-up the standby instance")
+
+	if err := standbyInstance.(abstract.WarmableInstance).Warmup(ctx, nil, logger); err != nil {
```

Comment:
Why not to use `worker.logger`?

Reply:
Because the `Warmup()` expects an `*echelon.Logger`, which is not what we use for the worker.